packages/extension-chakra-storefront/src/pages/checkout/partials/checkout-skeleton.jsx

```diff
@@ -27,13 +27,13 @@ const CheckoutSkeleton = () => {
                     </GridItem>
 
                     <GridItem py={6} px={[4, 4, 0]}>
-                        <Stack spacing={5}>
+                        <Stack gap={5}>
                             <Skeleton height="30px" width="50%" />
 
-                            <Stack spacing={5}>
+                            <Stack gap={5}>
                                 <Skeleton height="30px" width="65%" />
 
-                                <Stack w="full" py={4} borderY="1px" borderColor="gray.200">
+                                <Stack w="full" py={4} borderY="1px" borderColor="gray.200" borderStyle="solid">
```

Comment:
The borders are missing until I added borderStyle="solid", I'm not sure if there's some missing global theme defaults for the border styles? We didn't have to add that previously.

Reply:
Dumb question, which borders were you mentioning? I don't think we need global theme since stack does not usually have borders. We tend to use it a container for other elements